client/packages.go

```diff
@@ -37,6 +39,9 @@ type Package struct {
 	Version       string `json:"version"`
 }
 
+// Packages are collections of Package items
```

Comment:
That's unfortunate. I'll need to check whether golint has any way to disable such pedantic cases.

Not something for you to worry about, though. Fine for this to go in as-is for the time being.